app/controllers/idv/image_uploads_controller.rb

```diff
@@ -52,62 +17,13 @@ def create
 
     private
 
-    def image_form
-      @image_form ||= Idv::ApiImageUploadForm.new(
+    def image_upload_form
+      @image_upload_form ||= Idv::ApiImageUploadForm.new(
```

Comment:
Might not need to bother with memoization at this point, since we're only referencing it once? Or is it conventional to use memoization to ensure there's only one instance of a form being used in a class, regardless how often we're expecting the method to be called?

```suggestion
      Idv::ApiImageUploadForm.new(
```

Reply:
I don't think it hurts at all, but yeah we were referencing it multiple times but now it's just the once. ¯\_(ツ)_/¯